app/audit/models/PaymentAllocationsResponseAuditModel.scala

```diff
@@ -48,13 +48,26 @@ case class PaymentAllocationsResponseAuditModel(mtdItUser: MtdItUser[_],
     }
   }
 
+  private def getAllocationDescription(key: String): String = key match {
+    case "paymentAllocation.paymentAllocations.poa1.incomeTax" => "Payment allocated to Income Tax for payment on account 1 of 2"
+    case "paymentAllocation.paymentAllocations.poa1.nic4" => "Payment allocated to Class 4 National Insurance for payment on account 1 of 2"
+    case "paymentAllocation.paymentAllocations.poa2.incomeTax" => "Payment allocated to Income Tax for payment on account 2 of 2"
+    case "paymentAllocation.paymentAllocations.poa2.nic4" => "Payment allocated to Class 4 National Insurance for payment on account 2 of 2"
+    case "paymentAllocation.paymentAllocations.bcd.incomeTax" => "Payment allocated to Income Tax for remaining balance"
+    case "paymentAllocation.paymentAllocations.bcd.nic2" => "Payment allocated to Class 2 National Insurance for remaining balance"
+    case "paymentAllocation.paymentAllocations.bcd.vcnic2" => "Payment allocated to Voluntary Class 2 National Insurance for remaining balance"
+    case "paymentAllocation.paymentAllocations.bcd.nic4" => "Payment allocated to Class 4 National Insurance for remaining balance"
+    case "paymentAllocation.paymentAllocations.bcd.sl" => "Payment allocated to Student Loans for remaining balance"
+    case "paymentAllocation.paymentAllocations.bcd.cgt" => "Payment allocated to Capital Gains Tax for remaining balance"
+  }
```

Comment:
Please double check the values here. They look like the ones from Charge History table in Charge Summary page, rather than from the Payment Allocations page i.e. the `Payment allocated to ` part is not needed here.

Reply:
correct - changed 👍